pkg/trace/stats/otel_benckmark_test.go

```diff
@@ -9,17 +9,27 @@ import (
 	"testing"
 	"time"
 
-	"github.com/DataDog/datadog-agent/pkg/trace/config"
 	"github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/attributes"
 	"github.com/stretchr/testify/assert"
 	"go.opentelemetry.io/collector/component/componenttest"
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/ptrace"
 	semconv "go.opentelemetry.io/collector/semconv/v1.17.0"
 	"go.opentelemetry.io/otel/metric/noop"
+
+	"github.com/DataDog/datadog-agent/pkg/obfuscate"
+	"github.com/DataDog/datadog-agent/pkg/trace/config"
 )
 
 func BenchmarkOTelContainerTags(b *testing.B) {
+	benchmarkOTelContainerTags(b, false)
+}
+
+func BenchmarkOTelContainerTags_WithObfuscation(b *testing.B) {
+	benchmarkOTelContainerTags(b, true)
+}
+
+func benchmarkOTelContainerTags(b *testing.B, enableObfuscation bool) {
```

Comment:
These tests do not have any sql queries so obfuscation is just no-op. You will want to write a new benchmark on DB spans with sql queries.